l1-contracts/contracts/bridge/asset-tracker/AssetTracker.sol

```diff
@@ -144,6 +194,68 @@ contract AssetTracker is IAssetTracker, Ownable2StepUpgradeable, AssetHandlerMod
         chainBalance[chainToUpdate][_assetId] -= _amount;
     }
 
+    function handleChainBalanceIncreaseOnGateway(
+        uint256 _chainId,
+        bytes32 _canonicalTxHash,
+        bytes32 _baseTokenAssetId,
+        uint256 _baseTokenAmount,
+        bytes32 _assetId,
+        uint256 _amount
+    ) external {
+        if (_amount > 0) {
+            chainBalance[_chainId][_assetId] += _amount;
```

Comment:
do we need to care whether the chainId is the minter of assetId or baseTokenAssetId?

Reply:
true, I did not add that yet